fields/types/cloudinaryimages/CloudinaryImagesType.js

```diff
@@ -307,8 +296,20 @@ cloudinaryimages.prototype.updateItem = function (item, data, files, callback) {
 
 	async.map(values, function (value, next) {
 		if (typeof value === 'object' && 'public_id' in value) {
+			// Setting "autoCleanup" the property "remove:true" removes the file and resets the field
```

Comment:
@JedWatson finally getting back to this! Ill change the the comment to:
```
// If "autoCleanup" is enabled and the "remove" property is true for "value"
// than remove the file and reset the field
```
Does that clear it up?